Classes/Issues/Comments/IssueCommentSectionController.swift

```diff
@@ -75,6 +76,25 @@ AttributedStringViewIssueDelegate {
         return AlertAction(AlertActionBuilder { $0.rootViewController = weakSelf?.viewController })
             .share([url], activities: [TUSafariActivity()]) { $0.popoverPresentationController?.sourceView = sender }
     }
+    
+    func deleteAction() -> UIAlertAction? {
+        guard object?.viewerCanDelete == true, let number = object?.number else { return nil }
+        
+        return AlertAction.delete({ [weak self] _ in
+            let title = NSLocalizedString("Are you sure?", comment: "")
+            let message = NSLocalizedString("Deleting this comment is irreversible, do you want to continue?", comment: "")
```

Comment:
🤔, do we maybe want to add a feature were you can revert this until you leave the VC?

Reply:
Not sure if that many people would use it though.

Reply:
To be quite honest, I've only ever deleted like three comments in the several years I've used GitHub... I feel this is a pretty rare case that someone would then regret it